### PR TITLE
Update Plugin Doc for 5.18 Marketplace

### PR DESCRIPTION
**Pull request by JtheBAB**
The doc for the marketplace is stuck at 5.16. I tried to make it more visible that it is not only for 5.16 and distinguished the update possibilities prior 5.18 and 5.18+

### CODE REVIEW

File: source/administration/plugins.rst [hunk 1]
```diff
@@ -35,11 +35,15 @@ Once a plugin is installed (or pre-installed if shipped with Mattermost binary r
 3. Set **Enable Plugin** setting to ``True``. If this flag is not enabled, the plugin will not become active.
 4. Test out the plugin as needed.
 
-
 Upgrading Plugins
 ~~~~~~~~~~~~~~~~~
 
-In v5.16, the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**. In v5.18 (Dec 2019), you'll be able to upgrade a plugin on demand when a new version becomes available.
+In v5.16 and v5.17 the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in System Console > Plugin management.
```
Comment (amyblais) [inline source/administration/plugins.rst]:
```suggestion
In v5.16 and v5.17 the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**.
```

File: source/administration/plugins.rst [hunk 2]
```diff
@@ -35,11 +35,15 @@ Once a plugin is installed (or pre-installed if shipped with Mattermost binary r
...
 4. Test out the plugin as needed.
 
-
 Upgrading Plugins
 ~~~~~~~~~~~~~~~~~
 
-In v5.16, the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**. In v5.18 (Dec 2019), you'll be able to upgrade a plugin on demand when a new version becomes available.
+In v5.16 and v5.17 the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**.
+
+Upgrading Plugins with the Marketplace (5.18+)
```
Comment (lieut-data) [inline source/administration/plugins.rst]: Can we invert these paragraph blocks, to showcase v5.18 behaviour first? The v5.16/v5.17 behaviour will be less and less relevant over time, anyway.

**Comment by lieut-data** [comment]
Thanks for spotting this, @JtheBAB -- one comment above.

**Comment by JtheBAB** [comment]
@lieut-data

like that?

File: source/administration/plugins.rst [hunk 3]
```diff
@@ -35,16 +35,15 @@ Once a plugin is installed (or pre-installed if shipped with Mattermost binary r
 3. Set **Enable Plugin** setting to ``True``. If this flag is not enabled, the plugin will not become active.
 4. Test out the plugin as needed.
 
-Upgrading Plugins
-~~~~~~~~~~~~~~~~~
-
-In v5.16 and v5.17 the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**.
-
 Upgrading Plugins with the Marketplace (5.18+)
```
Comment (lieut-data) [inline source/administration/plugins.rst]: I'd propose dropng the v5.18 qualification here, and add `v5.16/v5.17` below.

File: source/administration/plugins.rst [hunk 4]
```diff
@@ -35,12 +35,15 @@ Once a plugin is installed (or pre-installed if shipped with Mattermost binary r
...
 Upgrading Plugins
 ~~~~~~~~~~~~~~~~~
 
-In v5.16, the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**. In v5.18 (Dec 2019), you'll be able to upgrade a plugin on demand when a new version becomes available.
+Upgrade a plugin on demand when a new version becomes available.
+
+Upgrading Plugins (prior to v5.18)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+In v5.16 and v5.17 the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and upload it in **System Console > Plugin management**.
```
Comment (justinegeffen) [inline source/administration/plugins.rst]:
```suggestion
In v5.16 and v5.17 the Marketplace only supports the installation of new plugins. To upgrade a plugin, you need to manually update it by downloading the binary file from the GitHub repository and then upload it in **System Console > Plugin Management**.
```

**Pull request merged**